python/llm/example/CPU/Applications/autogen/README.md

```diff
@@ -1,6 +1,8 @@
 ## Running AutoGen Agent Chat with BigDL-LLM on Local Models
-In this example, we use BigDL adapted FastChat to run [AutoGen](https://microsoft.github.io/autogen/) agent chat with 
```

Comment:
This example is adapted from the Official [AutoGen Teachablility tutorial](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb). We use a version of FastChat modified for BigDL to create a teachable chat agent with [AutoGen](https://microsoft.github.io/autogen/) that works with locally deployed LLMs. This special agent can remember things you tell it over time, unlike regular chatbots that forget after each conversation. It does this by saving what it learns on disk, and then bring up the learnt information in future chats. This means you can teach it lots of new things—like facts, new skills, preferences, etc..

In this example, we illustrate teaching the agent something it doesn't initially know. When we ask, `What is the Vicuna model`, it doesn't have the answer. We then inform it,`Vicuna is a 13B-parameter language model released by Meta`. We repeat the process for the Orca model, telling the agent, `Orca is a 13B-parameter language model developed by Microsoft. It outperforms Vicuna on most tasks.`. Finally, we test if the agent has learned by asking, `How does the Vicuna model compare to the Orca model?` The agent's response confirms it has retained and can use the information we taught it.